android/java/org/chromium/chrome/browser/playlist/settings/RadioButtonGroupPlaylistAutoSavePreference.java

```diff
@@ -0,0 +1,74 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.playlist.settings;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.RadioGroup;
+
+import androidx.preference.Preference;
+import androidx.preference.PreferenceViewHolder;
+
+import org.chromium.chrome.R;
+import org.chromium.components.browser_ui.widget.RadioButtonWithDescription;
+import org.chromium.components.browser_ui.widget.RadioButtonWithDescriptionLayout;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class RadioButtonGroupPlaylistAutoSavePreference
+        extends Preference implements RadioGroup.OnCheckedChangeListener {
+    static final int OPTIONS_SIZE = 4;
```

Comment:
I don't understand why here `OPTIONS_SIZE = 4` and not `3`.
I see array is allocated for 4 elements, but then only first 3 are set and then only first 3 are used in `onCheckedChanged`. 

Maybe  must be `OPTIONS_SIZE = 3` ?

Reply:
thanks